### ISSUE TITLE
Button styling

### ISSUE THREAD

**Issue by Mayamariaruth** [NoroffFEU/edu-gate#76]
## Tasks
- Update the button stylings on all pages (Desktop and mobile) to follow style guide below (There is currently no border color on secondary button for example)
- Update font sizes to match (The font sizes are different in different frames, ex. Results student and Add student buttons have different sizes, as well as the sizes in the same frame, ex. Add student)

### Design (from style guide)
Primary Button:
Background: #4CAF50 (primary color)
Text: White (#FFFFFF)
Hover: Darker shade of primary (#388E3C)
Border Radius: 6px
Padding: 10px 20px
Secondary Button:
Outline style with secondary color border (#2196F3) and text
Hover: Light secondary color background (#BBDEFB)

**Comment by QueenWoofie** [comment]
Added a component for the buttons and used that on both the desktop and mobile layout. 
Kept the primary font-size as 24px, but scaled it down to 16px on mobile on the pages where at least one button contained more than 8 characters. 

Adjusted the components to auto layout with hug, to account for the padding requirement of the style guide.

**Comment by Anne-Zwift** [comment]
@QueenWoofie I assumed that the buttons would follow the description of body text but maybe I'm wrong?
Headings: H1 - 32px, H2 - 24px, H3 - 20px
Body Text: 16px for paragraphs, 14px for labels and smaller text.

**Comment by pauiws** [comment]
Yes you have followed the style guide correctly, review complete :)

**Comment by QueenWoofie** [comment]
> [@QueenWoofie](https://github.com/QueenWoofie) I assumed that the buttons would follow the description of body text but maybe I'm wrong? Headings: H1 - 32px, H2 - 24px, H3 - 20px Body Text: 16px for paragraphs, 14px for labels and smaller text.

@Anne-Zwift When I started I used the main font size that was originally used on the desktop and just kept it as is. I think keeping it smaller, especially on the home page where the sign in and log in buttons are the primary targets would be too small, and I wanted to keep the sizing as consistant as possible. I'm gonna assume as the task was closed as completed, the sizing was fine.
